sirius/coding/templating.py

```diff
@@ -8,13 +8,13 @@
 # TODO apply scrubber library if wanted.
 # https://pypi.python.org/pypi/scrubber
 
-def default_template(raw_html, from_name):
+def default_template(raw_html, from_name, date=datetime.datetime.now()):
```

Comment:
`datetime.datetime.now()` will be evaluated only once, when the function is defined. Better to assign `None` as a default and test `if date is None:` in the function body to set a default

Reply:
Huh, I didn't know that! Good catch, and noted for future. Fixed in 044be3bf080b540fdba318d203edd6a327e56e82